neuralforecast/common/_modules.py

```diff
@@ -398,6 +398,59 @@ def forward(self, x):
         return self.embed(x)
 
 
+class FixedEmbedding(nn.Module):
+    def __init__(self, c_in, d_model):
+        super(FixedEmbedding, self).__init__()
+
+        w = torch.zeros(c_in, d_model).float()
+        w.require_grad = False
+
+        position = torch.arange(0, c_in).float().unsqueeze(1)
```

Comment:
Same here, I'd replace all the type conversions by instantiating with the correct type, avoiding a costly type conversion:
```suggestion
        position = torch.arange(0, c_in, dtype=torch.float32).unsqueeze(1)
```

Reply:
Done!